ral-facilities/scigateway-demo-plugin
# Added an example event being fired back to the parent

## Description
This is an event firing from the plugin back to the parent. It needs the redux middleware from the PR in the daaas-frontend repo in order to work.

## Testing instructions
Add a set up instructions describing how the reviewer should test the code


Functionality will be checked as part of the corresponding PR in the main repo

## Agile board tracking
connect to 23 in daaas-frontend
